core/src/client/lwm2m_static.c

```diff
@@ -131,6 +131,25 @@ AwaError AwaStaticClient_Init(AwaStaticClient * client)
     return result;
 }
 
+AwaError AwaStaticClient_SetLogLevel(AwaLogLevel level)
+{
+    AwaError result = AwaError_Unspecified;
+    AwaError AwaStaticClient_SetLogLevel(AwaLogLevel );
+    if (level >= AwaLogLevel_None && level <= AwaLogLevel_Debug)
+    {
+        Lwm2m_SetAwaLogLevel(level);
+        coap_SetLogLevel(level);
+
+        result = AwaError_Success;
+    }
+    else
+    {
+        result = AwaError_LogLevelInvalid;
```

Comment:
Is this error too specific? @david-antliff-imgtec @seank-img @chris-dewbery-imgtec